### DISCUSSION TITLE
Pass ref to useControl as prop instead of position

### DISCUSSION THREAD

**Discussion by maxwell-oroark** [uber/react-map-gl#2056] [Ideas]
I'm more or less trying to implement this mapbox [example](https://docs.mapbox.com/mapbox-gl-js/example/mapbox-gl-geocoder-outside-the-map) but with the react-map-gl useControl API:

Is there a way to pass a `ref` to a dom element to `useControl()` instead of the `position` argument?

**Comment by joel-daros** [comment]
Try this:
https://github.com/visgl/react-map-gl/discussions/2053#discussioncomment-4225133
